policies/coding-style.md

```diff
@@ -548,17 +549,17 @@ Functions can return values of many different kinds, and one of the
 most common is a value indicating whether the function succeeded or
 failed. Usually this is:
 
-    1: success
-    0: failure
+* 1: success
+* 0: failure
```

Comment:
On https://www.openssl.org/policies/technical/coding-style.html, these appear on a single line. Adding the bullet avoids that.